andela/AndelaEatsUI
# #160536360 Implement functionality for admin to view menus

#### What does this PR do?
- Adds functionality for admin to view all menus on the application

#### Description of Task to be completed?
- Remove in memory data used
- Make request to `mockapi` for menus
- Update test for new implementations

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-admin-view-menus-160536360`
- Run these commands `npm run bundle` and `npm run dev` to setup app
- Log in as an `Admin`, navigate to `admin dashboard`, select `Menus Icon` on the `side nav`.
- Ensure to be redirected to a page where you see listing of all menus
- Run the command `npm run test-dev` to ensure all tests are passing

#### What are the relevant pivotal tracker stories?
- [#160536360](https://www.pivotaltracker.com/story/show/160536360)

#### Screenshots (If Applicable)
<img width="1440" alt="screen shot 2018-10-10 at 5 11 03 pm" src="https://user-images.githubusercontent.com/32123313/46750377-b3a3ce00-ccaf-11e8-905d-b40dd086f392.png">
